### ISSUE TITLE
Next Steps

### ISSUE THREAD

**Issue by khalillechelt** [wbinnssmith/ungit#2]
Dear @reactivepod/reactivists & @reactivepod/owners:

First of all big ups to @wbinnssmith for putting this together!
Second of all how can we clean this up and get it shipped to Chrome App Store?

Has any of you tested this Chrome plugin?
Can we get some QA done here and maybe fix it up so that it can be published into the Chrome App Store?

I'd suggest a few todos here: 
- [x] change repo name to Gittens by popular vote
- [x] introduce `esnow` https://github.com/reactivepod/gittens/pull/3
- [x] introduce linting https://github.com/reactivepod/gittens/pull/5
- [ ] introduce tests
- [ ] fix any bugs that occur
- [ ] add icons
- [ ] publish to Chrome app store

Any thoughts, suggestions?

**Comment by yves-s** [comment]
Jepp, a name change would be better. I vote for gittens and I suggest **gittycat**

**Comment by wbinnssmith** [comment]
Gittens sounds great!

The idea behind the current name is that "git" has always meant an
unpleasant person and the extension "ungits" the page by turning them into
kittens.

Re: cleaning it up: yeah, it's missing a number of things like logos,
descriptions and other manifest-things. Also I attempted to keep the
dependencies minimal (no jquery, react, backbone or the like) but if it
gets any more complex it might be worth it. One small alternative that will
fit in nicely with the browserify builds is the stuff from @npm-dom

Of course, I've been meaning to move this to esnow since it seems like the
perfect candidate for that.

**Comment by khalillechelt** [comment]
> The idea behind the current name is that "git" has always meant an
> unpleasant person and the extension "ungits" the page by turning them into
> kittens.

Yeah I got that once @ollicle posted that Wiki link in the Slack chat. If you want to keep that name then you'd have to put an explanation in the README though I would say. I really just didn't get it at first.

Can you make a list of things that need to be done to clean the thing up?
Can you actually update my issue and put more check list items in there?

**Comment by ollicle** [comment]
My vote is for Gittens.

**Comment by andreruffert** [comment]
I've tested the Chrome plugins functionality real quick and everything looks :+1:

**Comment by khalillechelt** [comment]
Just changed the name to Gittens. It had the most :+1:  :smile:
